OpenRA.Mods.Common/Scripting/Properties/PlayerProperties.cs

```diff
@@ -39,18 +39,25 @@ public string Race
 			get
 			{
 				Game.Debug("The property `PlayerProperties.Race` is deprecated! Use `PlayerProperties.Faction` instead!");
-				return Player.PlayerReference.Faction;
+				return Player.Faction.InternalName;
 			}
 		}
 
 		[Desc("The player's faction.")]
-		public string Faction { get { return Player.PlayerReference.Faction; } }
+		public string Faction { get { return Player.Faction.InternalName; } }
```

Comment:
"real name" will break if we ever introduce translations, so lets keep the internal name.